File: trunk/Dobson/DobsonLibrary/Domain/Query.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DobsonLibrary.Domain
{
    public class Query
    {
        public string queryString { get; set; }
        public int queryID { get; set; }
        public int realmID { get; set; }
        public int userID { get; set; }

        public Query()
        {
            queryString = "";
            queryID = -1;
            realmID = -1;
            userID = -1;
        }
        public Query(string query, int realm, int userID)
        {
            queryString = query;
            realmID = realm;
            queryID = -1;
            this.userID = userID;
        }
    }
}

File:  ufmorpheus --username gycordero/OWLMorpheus/src/db/Query.java
package db;

public class Query {
	private int queryid;
	private String querystring;
	private int realmid;
	private Realm realm;	
	private Individual[] individual;
	
	public Query(int queryid, String querystring, int realmid) {
		this.queryid = queryid;
		this.querystring = querystring;
		this.realmid = realmid;
	}
	public void setRealm(Realm realm) {
		this.realm = realm;
	}
	public void setIndividual(Individual[] individual) {
		this.individual = individual;
	}
	public int getQueryid() {
		return queryid;
	}
	public String getQuerystring() {
		return querystring;
	}
	public Realm getRealm() {
		return realm;
	}
	public Individual[] getIndividual() {
		return individual;
	}
	public int getRealmid() {
		return realmid;
	}
}
